ui/admin/ember-cli-build.js

```diff
@@ -34,6 +34,9 @@ module.exports = function (defaults) {
     app.import('node_modules/clipboard/dist/clipboard.min.js');
   }
 
+  app.import('node_modules/codemirror/lib/codemirror.css');
```

Comment:
I unfortunately needed to import these files again. Should I just copy the contents and put them inside the rose styles?